File: android/entities/src/main/java/com/tiagohs/entities/image/ImageStyle.kt
package com.tiagohs.entities.image

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class ImageStyle(

    @SerializedName("width")
    var width: Int? = null,

    @SerializedName("height")
    var height: Int? = null,

    @SerializedName("resize")
    var resize: ImageResize? = null,

    @SerializedName("scale_type")
    var scaleType: String? = null
): Serializable

File: ios/HistoryCinema/HistoryCinema/Entities/LocalFiles/Image/ImageStyle.swift
//
//  ImageStyle.swift
//  HistoryCinema
//
//  Created by Tiago Henrique da Silva on 14/02/21.
//

import Foundation
import ObjectMapper

class ImageStyle: BaseLocalModel {
    var width: Int?
    var height: Int?
    var resize: ImageResize?
    var scaleType: String?
    
    required init?(map: Map) {
        super.init(map: map)
    }

    override func mapping(map: Map) {
        width       <- map["width"]
        height      <- map["height"]
        resize      <- map["resize"]
        scaleType   <- map["scale_type"]
    }
}

